### PR TITLE
stdout: vector of delegates, elf: custom section for symbols

### PR DESCRIPTION
**Pull request by fwsGonzo**
... adds support for backtrace and stack sampling on multiboot, including on GCE (I believe)

### CODE REVIEW

**Comment by alfred-bratterud** [comment]
@mnordsletten @fwsGonzo looks like the test fails due to objcopy being the wrong version or something. Any quick fix in sight?

**Comment by mnordsletten** [comment]
@alfred-bratterud I was planning on changing all the pull requests machines to Ubuntu 16.04 today, I will see if that fixes this specific issue.

**Comment by mnordsletten** [comment]
Ok to test

**Comment by mnordsletten** [comment]
The network test failed due to the dhcp test failing. I'll just run it again here.
Ok to test

File: .gitignore [hunk 1]
```diff
@@ -3,9 +3,6 @@
 *.o
 *.out
 *.log
-\#*
-\*.*
```
Comment (fwsGonzo) [inline .gitignore]: Atom behaves strangely when these exist in any gitignores, so we can't use them until fixed

**Comment by fwsGonzo** [comment]
test please

**Pull request merged**